tasks/kmt.py

```diff
@@ -651,15 +651,17 @@ def prepare(
     for d in domains:
         d.copy(ctx, paths.dependencies, "/opt/", verbose=verbose)
         d.copy(ctx, f"{paths.arch_dir}/opt/*", "/opt/", exclude="*.ninja", verbose=verbose)
-        d.run_cmd(
-            ctx,
-            f"! [ -f {btf_dir}/minimized-btfs.tar.xz \
-                && [ -d /opt/btf ] \
-                && cd /opt/btf/ \
-                && tar cJf minimized-btfs.tar.xz * \
-                && mkdir -p {btf_dir} \
-                && mv /opt/btf/minimized-btfs.tar.xz {btf_dir}/",
-        )
+        res = d.run_cmd("[ -f /sys/kernel/btf/vmlinux ]", warn=True)
+        if res is None or not res.ok:
+            d.run_cmd(
+                ctx,
+                f"! [ -f {btf_dir}/minimized-btfs.tar.xz ] \
```

Comment:
Could you add `[ -f /sys/kernel/btf/vmlinux ] || ` before this instead?

Reply:
That would work